lib/utils.js

```diff
@@ -18,6 +18,7 @@ function getRefValue (ref) {
 }
 exports.getRefValue = getRefValue
 
+const yamlSuffixRegex = /\.ya?ml/ // regex for matching .yaml/.yml files
```

Comment:
probably should be case-insensitive as well?

Reply:
i didn't see any special treatment to ref case anywhere, but i've updated the condition to disregard the file extension case